src/components/YearSelect/YearSelect.tsx

```
@@ -0,0 +1,113 @@
+      onChange?.(year)
+      setIsOpen(false)
+    },
+    [onChange],
+  )
+
+  const handleCloseDropdown = useCallback(() => {
+    setIsOpen(false)
+  }, [])
+
+  useEffect(() => {
+    if (!isOpen) return
```

ESLint

[error] Expected { after 'if' condition.

(curly)